test/mock/rtps/XMLProfileManager/fastrtps/xmlparser/XMLProfileManager.h

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Comment:
```suggestion
// Copyright 2022 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

I know this change is due to the delay in reviewing this PR. Sorry!